content/bn/docs/tasks/tools/included/kubectl-whats-next.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "এরপর কি?"
+description: "kubectl ইন্সটল করার পর কি হবে."
+headless: true
+_build:
+  list: never
+  render: never
+  publishResources: false
+---
+
+* [ইনস্টল করুন Minikube](https://minikube.sigs.k8s.io/docs/start/)
+* ক্লাস্টার তৈরি সম্পর্কে আরও জানতে দেখুন. [গাইড শুরু করা](/docs/setup/)
+* [আপনার অ্যাপ্লিকেশানটি কীভাবে লঞ্চ করবেন এবং প্রকাশ করবেন তা জানুন.](/docs/tasks/access-application-cluster/service-access-application-cluster/)
```

Comment:
```suggestion
* [আপনার অ্যাপ্লিকেশানটি কীভাবে লঞ্চ করবেন এবং প্রকাশ করবেন তা জানুন ।](/bn/docs/tasks/access-application-cluster/service-access-application-cluster/)
```

Reply:
Does that target page exist in the [dev-1.23-bn.1](https://github.com/kubernetes/website/tree/dev-1.23-bn.1) branch?

Reply:
Not yet @sftim  , thus I have removed the `bn` folder